components/inputs/input-date.js

```diff
@@ -232,7 +234,7 @@ class InputDate extends SkeletonMixin(FormElementMixin(LocalizeCoreElement(LitEl
 				@d2l-dropdown-focus-enter="${this._handleFocusTrapEnter}"
 				max-width="335"
 				min-height="${this._hiddenCalendarHeight}"
-				no-auto-fit
+				?no-auto-fit="${!mediaQueryList.matches}"
```

Comment:
Technically I think if `matches` changes (if the user resizes their window) the control isn't going to re-render. That's probably fine though since we really only do this when testing, but something to be aware of.